hera_cal/tests/test_smooth_cal.py

```diff
@@ -78,10 +78,19 @@ def test_freq_filter(self):
         gains[3,5] = 10.0
         wgts = np.ones((10,10),dtype=float)
         wgts[3,5] = 0
-        freqs = np.linspace(100.,200.,10)
+        freqs = np.linspace(100.,200.,10)*1e6
         ff = sc.freq_filter(gains, wgts, freqs)
         np.testing.assert_array_almost_equal(ff, np.ones((10,10),dtype=complex))
 
+        #test rephasing
```

Comment:
does this test that without the delay removal the gain smoothing fails? otherwise this isn't really testing that this method fixes the aforementioned problem...

Reply:
I have verified that it does fail without the delay removal.